common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -73,12 +73,12 @@ def build_xml(self, **kwargs):
         question.text = question_text
 
         # Add the response(s)
-        for i in range(0, int(num_responses)):
+        for __ in range(int(num_responses)):
```

Comment:
I was unaware of this particular convention (using `__` instead of i). Is this just because `i` is too short?